packages/core/src/api/firmware/utils/bootloaderHelper.ts

```
@@ -0,0 +1,223 @@
+      }
+
+      postTipMessage(device, postMessage, 'GoToBootloaderSuccess');
+      const isInBootloader = await checkDeviceToBootloader(device, connectId);
+      if (!isInBootloader) {
+        throw ERRORS.TypedError(HardwareErrorCode.FirmwareUpdateAutoEnterBootFailure);
+      }
+
+      if (DeviceModelToTypes.model_classic.includes(deviceType)) {
+        DevicePool.clearDeviceCache(uuid);
+      }
+      delete DevicePool.devicesCache[''];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)